content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -764,7 +765,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `UnknownVersionInteroperabilityProxy`: Proxy resource requests to the correct peer kube-apiserver when
   multiple kube-apiservers exist at varied versions.
   See [Mixed version proxy](/docs/concepts/architecture/mixed-version-proxy/) for more information.
-- `UserNamespacesStatelessPodsSupport`: Enable user namespace support for stateless Pods.
```

Comment:
Keep this line and revise it to mention that it has been renamed.

There are people using 1.25-1.27 releases and maybe they have enabled this for production workloads (stateless of course).

Reply:
Ok, I'll keep it.